### PR TITLE
Feature/nbpillai/portal-sidebar

### PR DESCRIPTION
**Pull request by nbpillai**
### Adminstrative Info
[Github Issue #90]

Make sure your branch name conforms to:
`<feature/staging/hotfix/...>/[username]/[3-4 word description seperated by dashes]`. Otherwise, please rename your branch and create a new PR.
### Changes
What changes did you make?
- styled and implemented side navigation bar
- linked all menu items to appropriate pages
- linked Laktaa logo to home page
- linked profile icon to profile page 
- styled menu items to change color according to page/active status
### Testing
How did you confirm your changes worked?
- added the side nav bar component to all pages 
- clicked on different menu bar items to ensure navigating to correct pages
- clicked on different menu bar items to ensure active status styling works 
- tested on different window sizes to check scroll feature and no cut off 
### Confirmation of Change
Upload a screenshot, if possible. Othwerwise, please provide instructions on how to see the change.

Screen Shot 2023-02-07 at 8 40 38 PM
Screen Shot 2023-02-07 at 8 40 10 PM
Screen Shot 2023-02-07 at 8 40 25 PM
Screen Shot 2023-02-07 at 8 40 31 PM

### CODE REVIEW

File: AdminPortal/src/pages/admin.tsx [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+import { Sidebar } from '../components/sidebar';
+
 export default function Admin() {
-  return <div>{'This is the admin page!'}</div>;
+  return (
+    <>
+      <main className={styles.main}>
+        {<div>{'This is the admin page!'}</div>}
+        <Sidebar currentPage={'/admin'} />
+      </main>
```
Comment (alexzhang1618) [inline AdminPortal/src/pages/admin.tsx:14]: Do we need curly braces around the admin page div? Additionally, I think it's better practice to put the sidebar before the content (also in profile.tsx, ratings.tsx, settings.tsx)

**Review by alexzhang1618 — CHANGES_REQUESTED** [review]
Changes look good overall! Just one small nitpick:

File: AdminPortal/src/pages/admin.tsx (hunk 1 above)
Comment (nbpillai) [reply AdminPortal/src/pages/admin.tsx:14]: fixed!

**Review by alexzhang1618 — APPROVED** [review]
Changes look good to me!

**Pull request merged**